extensions/wrapper/wrapper/src/test/java/de/sovity/edc/ext/wrapper/api/ui/pages/contract_agreement/ContractAgreementTransferApiServiceTest.java

```diff
@@ -95,6 +94,46 @@ void startTransferProcessForAgreementId(
         ));
     }
 
+    @Test
+    void startCustomTransferProcessForAgreementId(
+            ContractNegotiationStore store,
+            TransferProcessStore transferProcessStore
+    ) {
+        // arrange
+        var contractId = UUID.randomUUID().toString();
+        createContractNegotiation(store, COUNTER_PARTY_ADDRESS, contractId);
+
+        var customRequestJson = Json.createObjectBuilder()
+                .add(Prop.Edc.DATA_DESTINATION, Json.createObjectBuilder()
+                        .add(Prop.Edc.TYPE, "HttpData")
+                        .add(Prop.Edc.BASE_URL, DATA_SINK))
+                .add(Prop.Edc.PROPERTIES, Json.createObjectBuilder()
+                        .add("privateProperty", "privateValue"))
```

Comment:
Could you make sure, that the `https://w3id.org/edc/v0.0.1/ns/receiverHttpEndpoint` property can be set with an example url?